### PR TITLE
Feature/jp 23 imprint

### PR DESCRIPTION
**Pull request by EinSev**

### CODE REVIEW

File: frontend/pages/imprint.vue
Comment (Mathematti) [inline frontend/pages/imprint.vue:1]: > please report this to the e-mail address below

The email address is above

Comment (Mathematti) [reply frontend/pages/imprint.vue:1]: and also here
>  please let us know via the contact e-mail address below

Comment (Mathematti) [inline frontend/pages/imprint.vue:1]: Do we need a "Blattlinie"? We aren't a news company

Comment (Mathematti) [inline frontend/pages/imprint.vue:1]: Do we want to give credit to the icons here once again?

Comment (Mathematti) [inline frontend/pages/imprint.vue:1]: Maybe add an empty line here in between
image

File: frontend/pages/imprint.vue [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+                    </a>
+                </p>
+                <p>
+                    <b> <T key-name="imprint.dispute.title" />: </b><br />
+                    <T key-name="imprint.dispute.value" />:
+                    <a
+                        href="http://ec.europa.eu/od"
+                        class="hover:text-link hover:underline"
+                        >http://ec.europa.eu/od
+                    </a>
```
Comment (Mathematti) [inline frontend/pages/imprint.vue]: This link leads absolutely nowhere

File: frontend/pages/imprint.vue [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+                        <T key-name="imprint.contact.email.value" />
+                    </a>
+                </p>
+
+                <p>
+                    <b> <T key-name="imprint.legislation.title" />: </b>
+                    <a
+                        :href="t('imprint.legislation.value')"
+                        class="hover:text-link hover:underline"
+                    >
```
Comment (Mathematti) [inline frontend/pages/imprint.vue]: This link leads to http://localhost:3000/www.ris.bka.gv.at%E2%80%8C%E2%80%8C%E2%80%8C%E2%80%8C%E2%80%8C%E2%80%8C%E2%80%8D%E2%80%8C%E2%80%8C

**Review by Mathematti — CHANGES_REQUESTED** [review]

File: frontend/pages/imprint.vue [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+                    </a>
+                </p>
+                <p>
+                    <b> <T key-name="imprint.dispute.title" />: </b><br />
+                    <T key-name="imprint.dispute.value" />:
+                    <a
+                        href="http://ec.europa.eu/odr"
+                        target="_blank"
+                        class="hover:text-link hover:underline"
+                        >http://ec.europa.eu/odr
```
Comment (Mathematti) [inline frontend/pages/imprint.vue]: why no safe link? why not https://ec.europa.eu/odr

File: frontend/pages/imprint.vue [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+                        <T key-name="imprint.contact.phone.value" />
+                    </a>
+                    <br />
+                    <T key-name="imprint.contact.email.title" />:
+                    <a
+                        class="hover:text-link hover:underline"
+                        href="mailto:raven.burkard@journeyplanner.io"
+                    >
+                        <T key-name="imprint.contact.email.value" />
+                    </a>
```
Comment (Mathematti) [inline frontend/pages/imprint.vue]: image

File: frontend/pages/imprint.vue [hunk 5]
```diff
@@ -0,0 +1,104 @@
...
+                </p>
+
+                <p>
+                    <b> <T key-name="imprint.legislation.title" />: </b>
+                    <a
+                        href="www.ris.bka.gv.at"
+                        target="_blank"
+                        class="hover:text-link hover:underline"
+                    >
+                        <T key-name="www.ris.bka.gv.at" />
```
Comment (Mathematti) [inline frontend/pages/imprint.vue]: This still leads to localhost, I think you need to add the protocol

**Review by Mathematti — CHANGES_REQUESTED** [review]

File: frontend/pages/imprint.vue [hunk 6]
```diff
@@ -0,0 +1,104 @@
...
+            >
+                <T key-name="common.back" />
+            </button>
+        </div>
+        <div class="-mt-8 font-nunito text-text dark:text-input">
+            <div class="mx-5 mt-10 flex flex-col sm:mx-20 lg:mx-52 xl:mx-96">
+                <h4 class="text-center text-3xl font-bold">
+                    <T key-name="common.imprint" />
+                </h4>
+                <p class="mt-4">
```
Comment (stefll) [inline frontend/pages/imprint.vue]: noch ein mb-2

File: frontend/pages/imprint.vue [hunk 7]
```diff
@@ -0,0 +1,104 @@
...
+        <div class="flex justify-end">
+            <button
+                class="mr-10 mt-5 rounded-xl border-2 border-cta-border bg-input px-2 py-1 text-base font-semibold hover:bg-cta-bg dark:bg-input-dark dark:text-input dark:hover:bg-cta-bg-dark"
+                @click="$router.back()"
+            >
+                <T key-name="common.back" />
+            </button>
+        </div>
+        <div class="-mt-8 font-nunito text-text dark:text-input">
+            <div class="mx-5 mt-10 flex flex-col sm:mx-20 lg:mx-52 xl:mx-96">
```
Comment (stefll) [inline frontend/pages/imprint.vue]: mt-10 zu mt-7

File: frontend/pages/imprint.vue [hunk 8]
```diff
@@ -0,0 +1,104 @@
+<template>
+    <div>
+        <div class="flex justify-end">
+            <button
+                class="mr-10 mt-5 rounded-xl border-2 border-cta-border bg-input px-2 py-1 text-base font-semibold hover:bg-cta-bg dark:bg-input-dark dark:text-input dark:hover:bg-cta-bg-dark"
```
Comment (stefll) [inline frontend/pages/imprint.vue]: px-2 zu px-6

File: frontend/pages/imprint.vue [hunk 9]
```diff
@@ -0,0 +1,104 @@
...
+        </div>
+        <div class="-mt-8 font-nunito text-text dark:text-input">
+            <div class="mx-5 mt-10 flex flex-col sm:mx-20 lg:mx-52 xl:mx-96">
+                <h4 class="text-center text-3xl font-bold">
+                    <T key-name="common.imprint" />
+                </h4>
+                <p class="mt-4">
+                    <T key-name="imprint.info" />
+                </p>
+                <p>
```
Comment (stefll) [inline frontend/pages/imprint.vue]: mt-1

File: frontend/pages/imprint.vue [hunk 10]
```diff
@@ -0,0 +1,104 @@
...
+                    <b> <T key-name="imprint.operator.title" />: </b>
+                    <T key-name="imprint.operator.value" />
+                </p>
+                <p>
```
Comment (stefll) [inline frontend/pages/imprint.vue]: mt-1

File: frontend/pages/imprint.vue [hunk 11]
```diff
@@ -0,0 +1,104 @@
...
+                    <b> <T key-name="imprint.address.title" />: </b>
+                    <T key-name="imprint.address.value" />
+                </p>
+                <p>
```
Comment (stefll) [inline frontend/pages/imprint.vue]: mt-1

File: frontend/pages/imprint.vue (hunk 4 above)
Comment (stefll) [reply frontend/pages/imprint.vue]: ^

File: frontend/pages/imprint.vue [hunk 12]
```diff
@@ -0,0 +1,104 @@
...
+                    <T key-name="imprint.contact.email.title" />:
+                    <a
+                        class="hover:text-link hover:underline"
+                        href="mailto:raven.burkard@journeyplanner.io"
+                    >
+                        <T key-name="imprint.contact.email.value" />
+                    </a>
+                </p>
+
+                <p>
```
Comment (stefll) [inline frontend/pages/imprint.vue]: mt-1

File: frontend/pages/imprint.vue [hunk 13]
```diff
@@ -0,0 +1,104 @@
...
+                    <b> <T key-name="imprint.legislation.title" />: </b>
+                    <a
+                        href="www.ris.bka.gv.at"
+                        target="_blank"
+                        class="hover:text-link hover:underline"
+                    >
+                        <T key-name="www.ris.bka.gv.at" />
+                    </a>
+                </p>
+                <p>
```
Comment (stefll) [inline frontend/pages/imprint.vue]: mt-1

**Review by stefll — CHANGES_REQUESTED** [review]
Design-Review: Paar Abstände hinzugefügt und entfernt

File: frontend/pages/imprint.vue [hunk 14]
```diff
@@ -0,0 +1,106 @@
...
+            >
+                <T key-name="common.back" />
+            </button>
+        </div>
+        <div class="-mt-8 font-nunito text-text dark:text-input">
+            <div class="mx-5 mt-7 flex flex-col sm:mx-20 lg:mx-52 xl:mx-96">
+                <h4 class="mb-2 text-center text-3xl font-bold">
+                    <T key-name="common.imprint" />
+                </h4>
+                <p class="mt-1">
```
Comment (stefll) [inline frontend/pages/imprint.vue]: zu dem mt-1 beim p-tag das mb-2

File: frontend/pages/imprint.vue (hunk 11 above)
Comment (stefll) [reply frontend/pages/imprint.vue]: bitte bei dem p-tag eine mt-1 machen

**Review by stefll — CHANGES_REQUESTED** [review]
Design Review: Almost Done

**Review by Mathematti — APPROVED** [review]

**Review by stefll — APPROVED** [review]
Design Review: Everything looks fine

**Pull request merged**